### PR TITLE
Migrate home page and CSV parsing logic

### PR DESCRIPTION
**Pull request by colbyendres**
Currently, the home page after login is the index for `user`. This isn't useful, so change it to land on `schedules` instead.

### CODE REVIEW

**Comment by colbyendres** [comment]
I've also taken the liberty of migrating the CSV uploading/parsing logic to a separate service. CSV parsing doesn't really fit into the MVC paradigm so it makes sense to take it out of the controllers. This cleans up the code and admits a more modular way of reading in our datasets

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -9,8 +9,6 @@ class WelcomeController < ApplicationController
 
   def index
     return unless logged_in?
-
-    flash[:notice] = "Welcome back, #{@current_user.first_name}!" unless flash[:notice] || flash[:error]
-    redirect_to user_path(@current_user)
+    redirect_to schedules_path
```
Comment (colbyendres) [inline app/controllers/welcome_controller.rb:12]: PR title

File: app/controllers/schedules_controller.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'csv'
+# require_relative '../services/csv_handler'
```
Comment (Naviunni) [inline app/controllers/schedules_controller.rb]: remove comment

**Review by Naviunni — APPROVED** [review]
LGTM

**Pull request merged**